soco/core.py

```diff
@@ -1490,6 +1493,59 @@ def get_item_album_art_uri(self, item):
         else:
             return None
 
+    @only_on_master
+    def set_sleep_timer(self, sleep_time_seconds):
+        """Sets the sleep timer.
+
+        Args:
+            sleep_time_seconds (`int`): How long to wait before turning off
+                speaker in seconds. Maximum value of 86399
+
+        Returns:
+            bool: True if succesful, False otherwise
+
+        Raises SoCoException (or a subclass) upon errors, ValueError for
+            incorrect syntax.
+
+        """
+        if sleep_time_seconds and \
+           sleep_time_seconds != int(sleep_time_seconds):
+            raise ValueError('invalid sleep_time_seconds, must be integer \
+                value between 0 and 86399 inclusive')
+        if sleep_time_seconds:
```

Comment:
Sleep timer is disabled by calling ```set_sleep_timer``` with ```sleep_time_seconds``` set to 0. I've updated the doc in ```set_sleep_timer``` to reflect this and  ```get_sleep_timer``` to reflect that 0 will be returned when no sleep timer is set.